web/src/containers/Sidebar.test.js

```diff
@@ -9,32 +8,111 @@ import {
 } from './Sidebar';
 import { jumpTo } from '../actions/app';
 
+let mockPush;
+
+jest.mock('react-router-dom', () => {
+  mockPush = jest.fn();
+  return {
+    useHistory: jest.fn().mockReturnValue({ push: mockPush }),
+    useRouteMatch: jest.fn().mockReturnValue({ path: '---path---' })
+  };
+});
+
+global.scrollTo = jest.fn();
+
 describe('Sidebar component', () => {
   const props = {
     activities: [
-      { anchor: '#key1', key: 'key 1' },
-      { anchor: '#key2', key: 'key 2' }
+      { anchor: '#key1', key: 'key 1234' },
```

Comment:
Why this change? Because keys are 8-characters long, and the sidebar now relies on key length to figure out if an activity subsection is selected. So the test keys need to be 8 characters, too.